build.gradle

```diff
@@ -8,12 +8,16 @@ buildscript {
         maven {
             url 'https://maven.fabric.io/public'
         }
+        maven {
+            url "https://maven.mozilla.org/maven2"
+        }
     }
     dependencies {
         classpath "com.android.tools.build:gradle:${Versions.android_gradle_plugin}"
         classpath "com.google.android.gms:oss-licenses-plugin:${Versions.gms_oss_licenses_plugin}"
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${Versions.kotlin}"
         classpath "androidx.navigation:navigation-safe-args-gradle-plugin:${Versions.navigation}"
+        classpath "org.mozilla.components:tooling-glean-gradle:${Versions.mozilla_android_components}"
```

Comment:
Note that you'd want to add a dependency for unit tests as well. `testImplementation "org.mozilla.telemetry:glean-forUnitTests:${project.ext.glean_version}"`

Reply:
I'll add this when I add unit tests.